### ISSUE TITLE
Homily on the Nativity by Jacob of Serug

### ISSUE THREAD

**Issue by thea-m** [BetaMasaheft/Documentation#2358]
From @AaronButts https://github.com/BetaMasaheft/Documentation/issues/2351#issuecomment-1545624568

> Nevertheless, we will encounter a different case with the other homily of Jacob of Serugh mentioned in that excerpt from the EAe above, namely Jacob's Homily on the Nativity. Excerpts of it are found in the Hāymānota ʾabaw, and there is also a full translation. It is, I think, crucial to keep these things distinct: The excerpts of this homily in the Hāymānota ʾabaw are not taken from the full Ethiopic translation of the homily (as I understand the matter). Rather, the Arabic version of Jacob's homily on the nativity was translated into Ethiopic, and independently the Arabic Confession of the Fathers, in which there are the excerpts from Jacob's homily on the nativity, was translated into Ethiopic. In this way, the Ethiopic version of the entirety of the homily is really a different text from the excerpts in the Hāymānota ʾabaw. They both ultimately go back to the same Syriac homily by Jacob, but they have completely different histories and attestations in Ethiopic (as well as in Arabic). Does that make sense?

I found two records referring to a Homily on the Nativity by Jacob: LIT1186Baenta (created by conversion from the EAe index, referring to the quotations from this homily in the Hāymānota ʾabaw, not used) and LIT6605JacobNat, referring to a homily attested in EMM1763 (this would probably correspond to nr 13 in the EAe article on Jacob).

Is LIT6605JacobNat the independently translated version of the same homily that is attested in the form of quotations in the Hāymānota ʾabaw, or is it a completely different homily?

When this question is settled, we can decide on the encoding of this entire situation.

@AaronButts @MarcinKrawczuk

**Comment by AaronButts** [comment]
LIT6605JacobNat is *not* the independently translated version of the same homily that is attested in the form of quotations in the Hāymānota ʾabaw (i.e., LIT1186Baenta). It is more complicated, I am afraid. There are three "items" in play:

 LIT1186Baenta = quotations in the Hāymānota ʾabaw from Jacob's homily on ‘Nativity’ (B2 366 = A 201; ed. Bedjan 1902a, 720–774 = Bedjan 1902b, 108–62)

? = independent Ethiopic translation, found, e.g., in ms. Staatsbibliothek zu Berlin, Ms. or. fol. 117 ff. 120v-136v, of the same homily by Jacob, i.e., on ‘Nativity’ (B2 366 = A 201; ed. Bedjan 1902a, 720–774 = Bedjan 1902b, 108–62)

LIT6605JacobNat = independent Ethiopic translation, found, e.g., in EMML 1763, of a different Syriac homily by Jacob, also on 'Nativity' (B2 367 = A 202; ed. Bedjan 1902a, 775–790 = Bedjan 1902b, 163–78)

I am happy to clarify the sigla, etc., but this is exactly one of the problems that Ted and I are tackling: Most of the time, EAe entries do not adequately sort out the homilies attributed to Syriac authors. This is especially a problem for someone like Jacob, who often has multiple homilies on the same theme, so simply listing "Homily on Nativity" (as EAe tends to do) does not provide enough information. 

I am in the office if it helps to chat.

**Comment by MarcinKrawczuk** [comment]
Staatsbibliothek zu Berlin, Ms. or. fol. 117 ff. 120v-136v  = LIT6711JacSarGena (except I have doubts about folio numbers, didn't I mention it in an e-mail some time ago?)

**Comment by AaronButts** [comment]
Let me check my notes. Perhaps I made a mistake. I will work on it!

**Comment by AaronButts** [comment]
Indeed, CAe 6711 = independent Ethiopic translation, found, e.g., in ms. Staatsbibliothek zu Berlin, Ms. or. fol. 117 ff. 120v-136v, of the same homily by Jacob, i.e., on ‘Nativity’ (B2 366 = A 201; ed. Bedjan 1902a, 720–774 = Bedjan 1902b, 108–62). 

@thea-m, so there is no need to create a Clavis number for this one: It already exists. 

@MarcinKrawczuk thanks for finding it!
